prime-router/src/main/kotlin/fhirengine/engine/FHIRReceiverFilter.kt

```diff
@@ -434,6 +438,19 @@ class FHIRReceiverFilter(
                         )
                     )
 
+                    // add filter results
+                    emptyReport.filteringResults.add(
+                        ReportStreamFilterResult(
+                            receiver.fullName,
+                            db.fetchReportFile(queueMessage.reportId).itemCount,
+                            filterResult.failingFilter.filters.joinToString("\n"),
+                            emptyList(),
+                            AzureEventUtils.getIdentifier(bundle).value ?: "",
+                            filterResult.failingFilter.filterType,
+                            scope = ActionLogScope.report
```

Comment:
What's the difference if we put `item` here instead of `report`?

Reply:
I just thought report makes more sense because the whole report is getting filtered out. No issues or side effects if changed to item.

There's a separate ActionLog ReceiverItemFiltered (per item filter3d) that is scoped to item that we should maybe get rid of in favor of a summertime ReportStreamFilterResult.